test/functional/services/lib/web_element_wrapper/web_element_wrapper.ts

```diff
@@ -274,7 +323,9 @@ export class WebElementWrapper {
   public async getTagName<T extends keyof HTMLElementTagNameMap>(): Promise<T>;
   public async getTagName<T extends string>(): Promise<T>;
   public async getTagName(): Promise<string> {
-    return await this._webElement.getTagName();
+    return await this.retryCall(async function getTagName(wrapper: WebElementWrapper) {
+      return await wrapper._webElement.getTagName();
+    });
```

Comment:
Looks like `getTagName`, `getVisibleText`, and `getComputedStyle` could be DRY'd up.

Reply:
They could be, but then we'd lose the helpful function names in the stack traces, and I don't really think this is a place where repetition is causing an issue worth optimizing away.